frontend/integration-tests/data/caddy-docker.json

```diff
@@ -0,0 +1,31 @@
+{
+  "kind": "Pod",
+  "apiVersion":"v1",
+
+  "metadata": {
+        "name": "caddy-docker",
```

Comment:
Please fix the indentation in these files. (Converting to YAML would address this.)

Reply:
Yes, will do that